composer.json

```diff
@@ -21,7 +21,8 @@
         "php" : ">=7.1",
         "ext-json": "*",
         "psr/http-client": "^0.3.0",
-        "guzzlehttp/psr7": "^1.4"
+        "psr/http-factory": "^1.0",
+        "http-interop/http-factory-diactoros": "^1.0"
```

Comment:
This is broken. Requiring a specific http-factory implementation defeats the purpose of http-factory. This should be moved to `require-dev`.